### PR TITLE
✨Source Twilio: New streams Step

### LINKED ISSUE #26940: Source Twilio: new stream Execution Step
### What area the feature impact?

None

### Revelant Information

https://www.twilio.com/docs/studio/rest-api/v2/step#fetch-a-step-resource

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
Closes #26940

## Enhancement
New streams: 

- Step

## Recommended reading order
1. `source_twilio/source.py`
2. `source_twilio/streams.py`
3.  `source_twilio/schemas/*`

## 🚨 User Impact 🚨
No breaking changes

## Pre-merge Checklist
<details open><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter [Yes]

- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Issue acceptance criteria met
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [x] Documentation which references the generator is updated as needed

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-twilio/acceptance-test-config.yml [hunk 1]
```diff
@@ -46,6 +46,8 @@ acceptance_tests:
             bypass_reason: "very volatile data"
           - name: "usage_records"
             bypass_reason: "very volatile data"
+          - name: "step"
+            bypass_reason: "stream not filled yet"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-twilio/acceptance-test-config.yml:50]: @sajarin for GA connectors we must add data in the new stream. Only if there is a very specific condition where isn't possible we can skip them. Please take a look in the Twilio API and check how you can create some data points in the step.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

**Comment by sajarin** [comment]
@btkcodedev having some trouble creating a Step in our Twilio account. Any chance you could explain how to generate some Steps in a Flow on Twilio so we fetch those records from our account and update the expected records? Would be much appreciated.

**Comment by btkcodedev** [comment]
@sajarin, Sorry about that, the documentation only provides read operation through GET method and doesn't allow POST through cURL, 
Ref: https://www.twilio.com/docs/studio/rest-api/step

**Comment by btkcodedev** [comment]
@marcosmarxm Conflicts cleared, Version updated, Please continue further review

**Review by marcosmarxm — APPROVED** [review]
Thanks @btkcodedev

**Pull request merged**